velox/substrait/VeloxToSubstraitPlan.cpp

```diff
@@ -35,12 +35,28 @@ ::substrait::AggregationPhase toAggregationPhase(
       return ::substrait::AGGREGATION_PHASE_INTERMEDIATE_TO_RESULT;
     }
     default:
-      VELOX_NYI(
+      VELOX_UNSUPPORTED(
           "Unsupported Aggregate Step '{}' in Substrait ",
           mapAggregationStepToName(step));
   }
 }
 
+::substrait::SortField_SortDirection toSortDirection(
+    core::SortOrder sortOrder) {
+  if (sortOrder.isNullsFirst() && sortOrder.isAscending()) {
```

Comment:
this code might be easier to read if it is structures as 

```
if sortOrder.isNullsFirst()
   if sortOrder.isAscending()
   else
else
   if sortOrder.isAscending()
   else
```

Reply:
fix.